### ISSUE TITLE
/bg effects last when you leave the server and come back

### ISSUE THREAD

**Issue by zulater** [LorgeN/CrazyWars-Issues#129]
**Minecraft Username**
zulater

**What is this about?**
Islands, specifically /bg

**To Reproduce**
Drink a strength II potion and eat a gapple then immediately log off the server

**What happens?**
You can be logged off for 1 second or for 2 days but when you come back it's as if you just drank the strength II pot and ate the gapple

**What should happen?**
If time has passed you should not have these effects.  People are abusing this to be able to jump in and help without having to take the extra time to drink/gap before going out for a fight.

Possibly clear effects when switching to /bg?

**Screenshots/Videos**
https://youtu.be/wv3ElspoWTo

**Comment by LorgeN** [comment]
You are saying people are sitting in their main menu/in hub waiting for someone to be in trouble so they can log on, save 3 seconds and help them? What

**Comment by zulater** [comment]
No, you can visit another server or close mc down for days. The effects stay regardless of the passage of time.

**Comment by LorgeN** [comment]
Another server, yes, but that is how Minecraft works normally aswell. Your effects stop ticking once you disconnect.

**Comment by zulater** [comment]
Likely low priority, I'm just not a fan of the advantage you get from the ticking stopping when you leave.

**Comment by LorgeN** [comment]
It's hard to do much about. No information about last tick is stored.
